tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/Emitter.scala

```diff
@@ -462,4 +462,56 @@ object Emitter {
       value
     }
   }
+
+  private def makeSymbolRequirements(semantics: Semantics, esLevel: ESLevel): SymbolRequirement = {
+    import semantics._
+    import CheckedBehavior._
+
+    val factory = SymbolRequirement.factory("corejslib")
```

Comment:
Well, almost, actually. Only the `LongImpl` stuff does not come from `scalajsenv.js`. But `"emitter"`'s good too.